web/src/components/storage/DriveEditor.tsx

```diff
@@ -88,11 +88,16 @@ const SpacePolicySelector = ({ drive, driveDevice }: DriveEditorProps) => {
   const menuRef = useRef();
   const toggleMenuRef = useRef();
   const [isOpen, setIsOpen] = useState(false);
+  const navigate = useNavigate();
   const setSpacePolicy = useSetSpacePolicy();
   const onToggle = () => setIsOpen(!isOpen);
-  const onSpacePolicyChange = (spacePolicy: "keep" | "delete" | "resize") => {
-    setSpacePolicy(drive.name, spacePolicy);
-    setIsOpen(false);
+  const onSpacePolicyChange = (spacePolicy: "keep" | "delete" | "resize" | "custom") => {
+    if (spacePolicy === "custom") {
+      return navigate("/storage/space-policy/" + baseName(drive.name));
```

Comment:
This reminds me we had some conversations about what to use as id (basenames vs aliases vs something else). Is there something documented on that regard?

Reply:
It is still undecided. But I guess name/basename cannot be used for devices that are going to be created.